aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -12,3 +12,93 @@ proposals:
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/251"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - OPERATOR_BENEFICIARY
+  - name: change_commission_rate_delegation_pool
+    metadata:
+      title: "Change commission rates in delegation pools"
+      description: "AIP-50: This allows changing commission rates in delegation pools."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/249"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - COMMISSION_CHANGE_DELEGATION_POOL
+  - name: reduce_ident_size_max
+    metadata:
+      title: "Reduce the maximum identifier size"
+      description: "This reduces the maximum identifier size for (de)serializer of Move binary format"
+      discussion_url: "https://github.com/aptos-labs/aptos-core/pull/10502"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - LIMIT_MAX_IDENTIFIER_LENGTH
+  - name: introduce_single_sender_authenticator
+    metadata:
+      title: "Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts"
+      description: "AIP-55: Generalize Transaction Authentication and Support Arbitrary K-of-N MultiKey Accounts."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-55.md#aip-55---generalize-transaction-authentication-and-support-arbitrary-k-of-n-multikey-accounts"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - SINGLE_SENDER_AUTHENTICATOR
+  - name: allow_optional_fee_payer
+    metadata:
+      title: "Make Fee Payer Address Optional in Transactions Authenticator"
+      description: "AIP-53: Make Fee Payer Address Optional in Transactions Authenticator."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/257"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - FEE_PAYER_ACCOUNT_OPTIONAL
+  - name: enable_automatic_sponsored_account_creation
+    metadata:
+      title: "Automated Account Creation for Sponsored Transactions"
+      description: "AIP-52: Automated Account Creation for Sponsored Transactions."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/258"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - SPONSORED_AUTOMATIC_ACCOUNT_CREATION
+  - name: add_secp256k1_support
+    metadata:
+      title: "secp256k1 ECDSA for Transaction Authentication"
+      description: "AIP-49: secp256k1 ECDSA for Transaction Authentication."
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/247"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFlag:
+          enabled:
+            - Secp256k1ECDSAAuthenticator
+  - name: enable_aggreators_v2
```

Comment:
this will be in two phases
enable_aggregators_v2_api should go in v1.8, other flag in the followup, so let's change the name to enable_aggregators_v2_api and put AGGREGATOR_V2_API for the flag